bedrock/products/templates/products/vpn/variations/cta-b.html

```diff
@@ -0,0 +1,30 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "products/vpn/landing.html" %}
+
+ {% block hero_cta_change %}
+
+  {% set price = 'Get Mozilla VPN for ' + vpn_monthly_price(plan='12-month', lang=LANG) %}
+
+     {{ vpn_subscribe_link(
+        entrypoint=_utm_source,
+        link_text=price,
+        plan='12-month',
+        class_name='mzp-c-button mzp-t-xl ',
+        lang=LANG,
+        optional_parameters={
+          'utm_campaign': _utm_campaign
+        },
+        optional_attributes={
+          'data-cta-type': 'fxa-vpn',
+          'data-cta-text': 'Get Mozilla VPN 12-month',
+          'data-cta-position': 'pricing'
+        }
+      ) }}
+      <p>
+          <a href="#pricing" data-cta-text="Scroll to pricing">View all pricing plans</a>
```

Comment:
Add `data-cta-type="button"` and `data-cta-position="primary"`
```suggestion
          <a href="#pricing" data-cta-type="button" data-cta-text="Scroll to pricing" data-cta-position="primary">View all pricing plans</a>
```